packages/office-ui-fabric-react/src/components/Coachmark/Coachmark.types.ts

```diff
@@ -114,4 +116,39 @@ export interface ICoachmarkTypes extends React.Props<Coachmark> {
    * Beacon color two.
    */
   beaconColorTwo?: string;
+
+  /**
+   * Text to announce to screen reader / narrator when Coachmark is displayed
+   */
+  ariaAlertText?: string;
+
+  /**
+   * Ref for TeachingBubble
+   * @todo Fix interface here
+   */
+  teachingBubbleRef?: {
+    (component: ITeachingBubble | null): void;
+    current: ITeachingBubble | null;
+    value: ITeachingBubble | null;
```

Comment:
This seems like a weird flattened mixing of componentRef and  and createRef, did you mean to do this?

Reply:
Yes, not entirely sure what the right interface is.  If I set it to `teachingBubbleRef?: (component: ITeachingBubble | null) => void;`  then I don't have access to `this.props.teachingBubbleRef.current` in `Coachmark.tsx` (line 487).  Do you know what's the correct way to implement this?